styles/index.scss

```diff
@@ -24,9 +21,8 @@ html {
 }
 
 body {
-  font: 300 getFontSize(0) 'Open Sans', 'Century Gothic', sans-serif;
+  font: 300 getFontSize(0) 'Cabin', 'Century Gothic', sans-serif;
```

Comment:
Good catch, I was actually only importing Cabin in 400 and 600 (with italic versions of each) but forgot to update this... changing momentarily.